packages/sync/src/createClient.ts

```diff
@@ -14,10 +15,17 @@ import { PersistedData, PersistentStore } from "./PersistentStore";
 export const createClient = async (userConfig?: DataSyncConfig) => {
 
   const clientConfig = extractConfig(userConfig);
-  const cache = await buildStorage(clientConfig);
-  const link = buildLink(clientConfig);
-
-  return new ApolloClient({ link , cache });
+  const { cache, storage } = await buildStorage(clientConfig);
+  const link = buildLink(clientConfig, storage);
+  const options: ApolloClientOptions<NormalizedCacheObject> = {
+    link,
+    cache
+  };
+  const apolloClient = new ApolloClient<NormalizedCacheObject>(options);
+  const syncOfflineMutations = new SyncOfflineMutation(apolloClient, storage, "offline-mutation-store");
```

Comment:
"offline-mutation-store" - let's use config for this with default one pointing to this value.